modules/MiniPrograms/src/EasyAbp.WeChatManagement.MiniPrograms.Application/EasyAbp/WeChatManagement/MiniPrograms/Login/LoginAppService.cs

```diff
@@ -146,6 +148,45 @@ public virtual async Task UnbindAsync(LoginInput input)
             }
         }
 
+        [Authorize()]
+        public virtual async Task UnbindAsync(UnbindUserInput input)
+        {
+            await CheckUnbindPolicyAsync();
+
+            if (input.UserId != CurrentUser.Id)
```

Comment:
What if input.UserId is null?
https://github.com/EasyAbp/WeChatManagement/pull/28/files#diff-cf367485cad68cf4c031b06b6ca092df54473d457fc8f06c9422b78fa5021d45R17